src/endpoints/v1.es6.js

```diff
@@ -40,16 +43,18 @@ function baseGet(cache={}, uri, options, request, formatBody) {
 
   var key = uri + '?' + querystring.stringify(query);
 
-  if (headers.Authorization) {
-    cache = cache.authed;
-    key += '&auth=' + headers.Authorization;
-  } else {
-    cache = cache.unauthed;
-  }
+  if (options.useCache) {
```

Comment:
Add `&& cache`, and remove the default value from cache (`cache={}`) from baseGet and basePost.